views/pages/ResetPasswordPage.jsx

```diff
@@ -0,0 +1,261 @@
+import React, { Component } from 'react'
+
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import CardMedia from '@material-ui/core/CardMedia'
+import Typography from '@material-ui/core/Typography'
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
+import { routes } from '../routes/routes'
+import api from '../api'
+
+import basePathConfig from '../../server/configs/basePathConfig'
+
+const basePath = basePathConfig || ''
+
+class ResetPasswordPage extends Component {
```

Comment:
why are we creating this as a class component?

Reply:
This pr just makes sure that the pages work with client side routing, we're not up to the refactoring portion of this page.